sktime/utils/datetime.py

```diff
@@ -1,7 +1,10 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
+"""
+Times series functions related utilities.
```

Comment:
this should probably be "date/time format related utilities" or similar?

Reply:
I will change it to 'Time format related utilities'?